src/App.js

```diff
@@ -1,23 +1,35 @@
-import logo from './logo.svg';
-import './App.css';
-
+import React from 'react'
+import ProductList from './components/ProductList'
+import WishList from './components/WishList'
+import Home from './components/Home'
+import Cart from './components/Cart'
+import Productpage from './components/Productpage'
+import Login from './components/Login'
+import {Routes , Route}from 'react-router-dom'
+import './styles/App.css'
+import Filter from './components/Filter'
+import Sort from './components/Sort'
+import SignUp from './components/SignUp'
+import {PrivateRoute} from './util'
```

Comment:
import all the components from a single file ,that will look the code little neat.

Reply:
Sure will do it